app/jobs/people/create_roles_job.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+#  Copyright (c) 2023, Schweizer Alpen-Club. This file is part of
+#  hitobito and licensed under the Affero General Public License version 3
+#  or later. See the COPYING file at the top-level directory or at
+#  https://github.com/hitobito/hitobito.
+#
+class People::CreateRolesJob < RecurringJob
+
+  run_every 15.minutes
+  Error = Class.new(StandardError)
+
+  def perform
+    future_roles.find_each { |role| convert(role) }
+  end
+
+  private
+
+  def future_roles
+    FutureRole.where('convert_on <= ?', Time.zone.today).order(:convert_on)
```

Comment:
Bitte gemäss Spezifikation noch diejenigen FutureRoles ausschliessen und im UI markieren, welche in der Vergangenheit fehlgeschlagen sind

Reply:
Spez wurde angepasst 

`wir gehen davon aus das in diesem Fall alle 15min wenn der Job läuft eine Exception geraised wird die wir dann im sentry sehen bis das Problem behoben wurde.`